### ISSUE TITLE
Blighted Herd - Chapter 14 PQ (Greenskins)

### ISSUE THREAD

**Issue by Jerkling** [WarEmu/WarBugs#10898]
The Greenskins Chapter 14 Hard PQ "Blighted Herd" shows as for Ranks 24-25 when it should actually show Ranks 28-29

![vladolfr_003](https://user-images.githubusercontent.com/33235244/33150842-b0af22f8-cfd5-11e7-8554-0f068e227ebb.jpg)

**Comment by sioding** [comment]
As the Wiki said: https://wiki.returnofreckoning.com/index.php/Public_Quests

> Blighted Herd Hard Public Quest Destruction Greenskin Chapter 14 | RANKS 24-25 | Advanced Rewards

**Comment by Lheana** [comment]
Exact. Nevertheless it is client related, so not fixable yet. We can close this ticket until we can fix it.
Note: I am aware it is not the only PQ that does not show his correct level, so no need to report the others. I will fix it asap.
